Javadoc warnings

Let's get rid of these warnings...

> Task :javadoce/alessio/dati/onnxruntime-genai-main/build/Linux/Release/wheel
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:32: warning: no comment
    bool,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:33: warning: no comment
    float16,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:24: warning: no comment
    float32,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:34: warning: no comment
    float64,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:28: warning: no comment
    int16,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:29: warning: no comment
    int32,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:30: warning: no comment
    int64,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:26: warning: no comment
    int8,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:31: warning: no comment
    string,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:27: warning: no comment
    uint16,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:35: warning: no comment
    uint32,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:36: warning: no comment
    uint64,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:25: warning: no comment
    uint8,
    ^
/home/alessio/dati/onnxruntime-genai-main/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java:23: warning: no comment
    undefined,
    ^
14 warnings

--- a/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java
+++ b/src/java/src/main/java/ai/onnxruntime/genai/Tensor.java
@@ -20,19 +20,33 @@ public final class Tensor implements AutoCloseable {
   // https://github.com/onnx/onnx/blob/159fa47b7c4d40e6d9740fcf14c36fff1d11ccd8/onnx/onnx.proto#L499-L544
   /** Element types that correspond to OnnxRuntime supported element types. */
   public enum ElementType {
+    /** Undefined element type. */
     undefined,
+    /** 32-bit IEEE 754 floating point. */
     float32,
+    /** 8-bit unsigned integer. */
     uint8,
+    /** 8-bit signed integer. */
     int8,
+    /** 16-bit unsigned integer. */
     uint16,
+    /** 16-bit signed integer. */
     int16,
+    /** 32-bit signed integer. */
     int32,
+    /** 64-bit signed integer. */
     int64,
+    /** UTF-8 encoded string. */
     string,
+    /** Boolean value (true or false). */
     bool,
+    /** 16-bit IEEE 754 floating point (half precision). */
     float16,
+    /** 64-bit IEEE 754 floating point (double precision). */
     float64,
+    /** 32-bit unsigned integer. */
     uint32,
+    /** 64-bit unsigned integer. */
     uint64,
   }
 

